tgdb/include/include.php

```diff
@@ -245,4 +245,4 @@ function generateDurationFromDates($start, $end) {
 
 
 
-?>
\ No newline at end of file
+?>
```

Comment:
I only noticed this after merge

in php files having a new line after the closing statement can break things.

I'll have to fix the github editor adding them.